smart-metastore/src/test/java/org/smartdata/metastore/dao/accesscount/TestFileAccessManager.java

```diff
@@ -83,6 +95,55 @@ public void testSaveAccessCounts() throws MetaStoreException {
         fileAccessInfos.getItems());
   }
 
+  @Test
+  public void testSearchByIds() throws MetaStoreException {
+    List<Long> ids = Arrays.asList(1L, 2L);
+    long currentTimeMillis = System.currentTimeMillis();
+    insertFileAccessCounts(currentTimeMillis);
+    testSearch(FileAccessInfoSearchRequest.builder().ids(ids).build(), ids.toArray(new Long[0]));
+  }
+
+  @Test
+  public void testSearchByPath() throws MetaStoreException {
+    long currentTimeMillis = System.currentTimeMillis();
+    insertFileAccessCounts(currentTimeMillis);
+    testSearch(FileAccessInfoSearchRequest.builder()
+            .pathLike("/file3")
+            .build(),
+        Collections.singletonList(3L).toArray(new Long[0]));
+    testSearch(FileAccessInfoSearchRequest.builder()
+            .pathLike("/file")
+            .build(),
+        Arrays.asList(1L, 2L, 3L).toArray(new Long[0]));
```

Comment:
nit: you can use varargs instead of constructing arrays
```java
testSearch(FileAccessInfoSearchRequest.builder()
             .pathLike("/file")
             .build(),
         1L, 2L, 3L);
```

Reply:
fixed